http/swagger.yaml

```diff
@@ -51,3 +52,65 @@ paths:
           description: Message discarded due to invalid or missing content type.
         500:
           description: Unexpected server-side error occurred.
+definitions:
+  SenML:
```

Comment:
I would call this one SenML_record

Reply:
Ok. Got the point. Do we use snake_case or CamelCase?

Reply:
Looks like you should use `senml` and `senmlRecord`

Reply:
I opted here for SenMLRecord and SenMLArray, because we are using capitals for the first letter of definitions.